src/common/partialRestore.c

```diff
@@ -0,0 +1,154 @@
+#include "build.auto.h"
+#include "common/type/json.h"
+#include "common/type/list.h"
+#include "config/config.h"
+#include "partialRestore.h"
+#include "storage/helper.h"
+
+typedef struct DataBase
+{
+    // It must be the first element since it is used for sorting
+    Oid dbOid;
+    List *tables;
+} DataBase;
+
+typedef struct Table
+{
+    Oid spcNode;
+    Oid relNode;
+} Table;
+
+static List *filterList = NULL;
```

Comment:
Maybe make this variable static inside a function? It should not be used outside of it anyway.

Reply:
Done